ec-europa/eubfr-data-lake
# feat(instance-stopper): initiate service - EUBFR-189

# PR description

- Based on working lambda function from @raduchiriac 
- Limited permissions though
- Instances need to have `automatedStop` tag with `true` value

Only stopping to save time, possibly could start automatically as well, there are [other methods](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/ec2-example-managing-instances.html) of course.

To test without waiting until 17h:
- yarn deploy either in root or in service folder
- invoke the lambda function through the console with any event structure
- check ec2 console for having the instance terminated

## QA Checklist

When you add a new ETL/producer, please check for the following:

* [ ] Producer's secrets are stored safely
* [ ] Ensure the ETL is added to the corresponding `scripts/` for automated deployment and deletion 
* [ ] There is at least 1 unit test with a jest snapshot for the transform function of the ETL
* [ ] Update the file PRODUCERS_DATA_AVAILABILITY_GRID.md indicating which fields are available in source data of ETL
* [ ] Ensure there is (flow/jsdocs) documentation in the `tranform.js` file which is to be used for automated documentation
* [ ] Generate the necessary documentation pages for the new ETL by `yarn docs:md`
